.github/workflows/main.yaml

```diff
@@ -68,29 +68,37 @@ jobs:
       ]
     uses: ./.github/workflows/test-nx-project-e2e.yaml
     with:
-      nx_project: administration
-      nx_app_port: 4001
       image_url: "ghcr.io/bcgov/cas-admin-frontend"
       is_nx_affected: ${{contains(needs.check-nx-affected.outputs.NX_AFFECTED_E2E, 'administration') || github.ref_name == 'develop' || github.ref_name == 'main' || github.head_ref == 'develop' || github.head_ref == 'main'}}
+      nx_app_port: 4001
+      nx_project: administration
     secrets: inherit
   test-coam-e2e:
     needs: [build-backend, build-dashboard-e2e, build-coam, check-nx-affected]
     uses: ./.github/workflows/test-nx-project-e2e.yaml
     with:
-      nx_project: coam
-      nx_app_port: 7000
       image_url: "ghcr.io/bcgov/cas-coam-frontend"
       is_nx_affected: ${{contains(needs.check-nx-affected.outputs.NX_AFFECTED_E2E, 'coam') || github.ref_name == 'develop' || github.ref_name == 'main' || github.head_ref == 'develop' || github.head_ref == 'main'}}
+      nx_app_port: 7000
+      nx_project: coam
     secrets: inherit
```

Comment:
A lot of these changes are me just neurotically putting things in alphabetical order 😅